projects/blockchain-lottery-and-gambling.md

```diff
@@ -23,4 +23,12 @@ Our first story is [here](https://trello.com/c/FjEsJj5m/13-first-story-users-can
 
 ### Version Control
 
-Our code is hosted on [GitHub](https://github.com/HarrisonGregg/BlockchainGambling).
\ No newline at end of file
+Our code is hosted on [GitHub](https://github.com/HarrisonGregg/BlockchainGambling).
+
+### Automated Testing and Continuous Integration [![Build Status](https://travis-ci.org/HarrisonGregg/BlockchainGambling.svg)](https://travis-ci.org/HarrisonGregg/BlockchainGambling)
+
+On upload to GitHub, [Travis CI](https://travis-ci.org/HarrisonGregg/BlockchainGambling) automatically runs our Django unit tests and uploads all changes to Heroku if all tests pass.
```

Comment:
Technically, _uploads all changes_ is incorrect. It _pushes_ all changes or _deploys_ them. It sounds subtle, but using clear terminology helps everybody speak the same language :)